packages/playground/src/__tests__/api/documentsApi.uint.spec.ts

```diff
@@ -745,4 +745,50 @@ describe('Documents unit test', () => {
     const resp = await client.documents.preview.temporaryLink(1);
     expect(resp.temporaryLink).toEqual(link);
   });
+
+  describe('classifiers', () => {
+    test('create', async () => {
+      nock(mockBaseUrl)
+        .post(new RegExp('/documents/classifiers'), {
+          items: [{ name: 'test' }],
+        })
+        .once()
+        .reply(200, {
+          items: [{ name: 'test' }],
+        });
+      const resp = await client.documents.classifiers.create([
+        { name: 'test' },
+      ]);
+      expect(resp[0].name).toEqual('test');
+    });
+    test('list by ids', async () => {
+      nock(mockBaseUrl)
+        .post(new RegExp('/documents/classifiers'), {
```

Comment:
I think it should have `/byids` at the end here? https://docs.cognite.com/api/playground/#operation/documentsListClassifiersByids

If I understand https://github.com/nock/nock#specifying-path correctly I also think this RegExp pattern both matches `/documents/classifiers` as well as `/documents/classifiers/byds`. I'd recommend only using RegExp pattern where it makes sense to do so, so keeping a plain string in this case.